### ISSUE TITLE
value/label pairs don't exist in datalist

### ISSUE THREAD

**Issue by murtaugh** [mithra62/ee-datalist-ft#1]
I didn't know this until I gave this field a try, but apparently the datalist field type doesn't support option labels (see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist).

The screenshot below is what you get in Safari when you use the value/label pairs option when setting up the field:

<img width="331" alt="image" src="https://user-images.githubusercontent.com/308752/148283630-fa2c78ca-612a-47db-8b6f-e9131221c2cd.png">

**Comment by murtaugh** [comment]
This is a strange disappointment.

**Comment by murtaugh** [comment]
But after thinking about it for 5 seconds, it makes sense, because any human-entered data would not be in the same label/value format.

**Comment by eric-lamb62** [comment]
Yeah, it's a weird one, isn't it? Am open to suggestions on how best to handle that though.

**Comment by murtaugh** [comment]
Is there a way to hide the "Value/Label Pairs" option? I think that's all that would need to be done.

**Comment by murtaugh** [comment]
But just knowing not to use that is enough for me at the moment.

**Comment by eric-lamb62** [comment]
> Is there a way to hide the "Value/Label Pairs" option? I think that's all that would need to be done.

No, that's an internal EE thing :(
